packages/app-bridge/src/AppBridgeTheme.spec.ts

```diff
@@ -152,8 +152,8 @@ describe('AppBridgeThemeTest', () => {
         HttpClient.get = mockHttpClientGet;
 
         const mappedDocumentGroups = [
-            DocumentGroupDummy.with(123, [DocumentDummy.with(123), DocumentDummy.with(456)]),
-            DocumentGroupDummy.with(456, [DocumentDummy.with(123), DocumentDummy.with(456)]),
```

Comment:
Please go with the previous version of `with`, it allows a deeper customization of `documents`/`DocumentDummy`.
This is also the way we do it in the backend so it is more aligned.